pkg/cloud/ovirt/clients/machineservice.go

```diff
@@ -360,10 +371,53 @@ func  (is *InstanceService) FindVirtualMachineIP(id string,excludeAddr map[strin
 
 				if hasAddress {
 					klog.Infof("ovirt vm id: %s , found usable IP %s", id, ipres)
-					return ipres,nil
+					return ipres, nil
 				}
 			}
 		}
 	}
 	return "", fmt.Errorf("coudlnt find usable IP address for vm id: %s", id)
 }
+
+func (is *InstanceService) getAffinityGroups(cID string, agNames []string) (ag []*ovirtsdk.AffinityGroup, err error) {
```

Comment:
docString

Reply:
I think the name explains the function pretty good and since it is just internal I don't think that a docString is needed